docs/2_3_fmu_distribution.adoc

```diff
@@ -222,3 +225,10 @@ Furthermore, in order that the C compiler can check for consistent function argu
 It would therefore be counter-productive (unsafe) if this header file was present. +
 These header files are not included in the `binaries` directory, since they are already utilized to build the target simulator executable.
 The version number of the header file used to construct the FMU can be deduced via attribute `fmiVersion` in file `modelDescription.xml` or via function call `fmi3GetVersion`.]_
+
```

Comment:
We don't currently use *bold* and UPPERCASE in the documentation. IMHO this excessive formatting make the text less readable.

Reply:
It is the RFC 2119 keyword formatting that we introduced at the design meeting, which is the usual way the keywords are highlighted (in document formats that offer bold). But these can be changed to just UPPERCASE (or whatever other format that makes them stick out) once we do the document regularization. Only in the specified formatting do they get their normative meaning.

Reply:
What would be the problem if we always assume the normative meaning of the keywords (regardless of formatting)?

Reply:
The idea behind RFC 2119 is to be explicit, i.e. to prevent accidental use of words like can, may, etc., becoming normative, and forcing the authors to think carefully about each required behavior.